seq_logger.js

```diff
@@ -1,13 +1,10 @@
 "use strict";
 
-let http = require('http');
-let https = require('https');
-let url = require('url');
-
+const NodeBlob = require('buffer').Blob
+const GlobalBlob = Blob !== undefined ? Blob : NodeBlob
 const HEADER = '{"Events":[';
 const FOOTER = "]}";
-const HEADER_FOOTER_BYTES = Buffer.byteLength(HEADER, 'utf8') + Buffer.byteLength(FOOTER, 'utf8');
-
+const HEADER_FOOTER_BYTES = (new GlobalBlob([HEADER])).size + (new GlobalBlob([FOOTER])).size
 
```

Comment:
`fetch` can be polyfilled by referencing node-fetch@2 and then something like: 

`const fetchApi = typeof(fetch) === 'undefined' ? require('node-fetch') : fetch;`

then it works in node 16